src/PointwiseFunctions/AnalyticSolutions/GrMhd/KomissarovShock.hpp

```diff
@@ -0,0 +1,257 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <limits>
+#include <string>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/IdealFluid.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+// IWYU pragma: no_include <pup.h>
+
+/// \cond
+namespace PUP {
+class er;  // IWYU pragma: keep
+}  // namespace PUP
+/// \endcond
+
+namespace grmhd {
+namespace Solutions {
+
+/*!
+ * \brief A one-dimensional shock solution for an ideal fluid in Minkowski
+ * spacetime
+ *
+ * This solution consists of a left state for \f$x<0\f$ and a right state for
+ * \f$x>=0\f$, each with constant fluid variables. The interface between these
```

Comment:
`\ge`

Reply:
There have been several discussions, and I'm afraid I don't know which one you are talking about.  No one has ever bothered to propose guidelines or a review policy based on any of them, so you can't really expect people to follow ideas from whatever discussion you are thinking of.

In the lastest discussion I know of (which you were not part of) I was actually _encouraged_ to hold off on reviewing PRs until a substantial amount of review had happened.  So everyone has different expectations and I can't follow all of them.

Reply:
@nilsleiffischer @wthrowe is working on his thesis right now so Larry and I have intentionally left Will until the end so that the review is easier for him. Hopefully you can understand that.

I also agree with Will that such a discussion of PR review processes first needs to happen in the SpECTRE call and we need to have them in writing. I'm assuming that your (@nilsleiffischer) comment was merely meant to point something out to Larry and I. Let's not do that going forward or point it out offline so we don't end up with these sorts of possibly confusing conversations :)

Reply:
@nilsdeppe please stop trying to explain to people why I'm doing things or what I'm talking about.  Your explanation wasn't even close to the mark in this case.

What I was refering to was the request that we allow a variety of people to do initial reviews of PRs so that they can gain more experience with the review process.  The more experienced reviwers were encouraged to hold off on doing detailed reviews if there was an active review going on.